### DISCUSSION TITLE
Object detection with feature columns (How to with YOLOv5?)

### DISCUSSION THREAD

**Discussion by MLDavies** [ultralytics/yolov5#6870] [Q&A]
Hi, does anyone have an example (if it exists yet) or ideas for how to include additional data/metadata as a feature column(s) with images to the YOLOv5? I'm thinking of something like:  https://www.tensorflow.org/tutorials/structured_data/feature_columns

**Comment by glenn-jocher** [comment]
@MLDavies your example is regression/classification on tabular (spreadsheet) data. This is unrelated to vision AI.
